### DISCUSSION TITLE
Forge Test and Forge Coverage not working

### DISCUSSION THREAD

**Discussion by keeping2Code** [foundry-rs/foundry#7889] [Q&A] [answered]
I'm having issues running forge test/coverage, Foundry isn't recognizing any of my tests. No matter what changes I make to any of my tests it just says compiler run successful, and when I run forge coverage it keeps coming back 0% on everything. I also used 'forge inspect <test contract> methods' and I'm able to see all of the functions in the test contract but when I try and run tests -- nothing. I'm using Foundry 0.2.0 so what do I need to do to fix this?

**Comment by onbjerg** [comment]
Can you post the exact output of `forge --version` and also a snippet of a test contract?

**Reply by keeping2Code** [reply, marked as answer]
Thanks for responding I actually figured out what it was. I was importing Ownable from Openzeppelin and having it initialize an initialOwner in the constructor, once I deleted that in my contracts everything was working as normal!
